src/webpanel.ts

```diff
@@ -0,0 +1,78 @@
+import * as vscode from 'vscode';
```

Comment:
missing header

Reply:
Added MIT license header and mention that we got this from the kubernetes tooling project